articles/container-registry/container-registry-repository-scoped-permissions.md

```diff
@@ -139,6 +142,70 @@ The output shows details about the token. By default, two passwords are generate
 > [!NOTE]
 > To regenerate token passwords and expiration periods, see [Regenerate token passwords](#regenerate-token-passwords) later in this article.
 
+### How to use scope maps to define and assign permissions for multiple repositories
+
+Scope maps allow for the use of a wildcard character to define and grant similar permissions for multiple repositories sharing a common prefix. Repositories specific permissions and repositories with a wildcard character can be used in the same scope map. This will allow flexibility to manage permissions for multiple repositories in a single scope map.
+
+These can be created when a scope map is created and assigned to a token, or when a token is created and directly assigned to a repository.
+
+The following example creates a scope map with a wildcard character and then assigns it to a token.
+
+```azurecli
+az acr scope-map create --name MyScopeMapWildcard --registry myregistry \
+  --repository samples/* \
+  content/write content/read \
+  --description "Sample scope map with wildcards"
+az acr token create --name MyTokenWildcard \
+  --registry myregistry \
+  --scope-map MyScopeMapWildcard
+```
+
+The following example creates a token with a wildcard.
+
+```azurecli
+ az acr token create --name MyTokenWildcard --registry myregistry \
+  --repository samples/* \
+  content/write content/read \
+```
+
+The wildcard permissions are additive, which means that when a specific repository is accessed, the resulting permissions will include the permissions for all the scope map rules that match the wildcard prefix.
+
+In this example, if a token is assigned to a scope map with the following repositories defined:
+
+  |Repository  |Permission  |
+  |---------|---------|
+  |`sample/*`    | `content/read`  |
+  |`sample/teamA/*`    | `content/write`  |
+  |`sample/teamA/projectB`    | `content/delete`  |
+
+The token would have `[content/read, content/write, content/delete]` when accessing repository `sample/teamA/projectB`. And will have only `[content/read, content/write]` when accessing repository `sample/teamA/projectC`.
```

Comment:
The token is assigned a scope map to grant `[content/read, content/write, content/delete]` permissions for accessing repository `sample/teamA/projectB`. However, when the same token is used to access the `sample/teamA/projectC` repository, it only has `content/read` and `content/write` permissions.